packages/pyright-internal/src/languageService/indentationUtils.ts

```diff
@@ -30,6 +32,8 @@ import {
     TokenType,
 } from '../parser/tokenizerTypes';
 
+export const TabSize = 8;
```

Comment:
For consistency with the rest of the code base, please make constant names lowercase.

Reply:
okay